### PR TITLE
update small microwave satellite converters to naming conventions

### LINKED ISSUE #1185: update small satellite converters for naming conventions
## Description

the TROPICS, TEMPEST and COWVR converters should be updated to the new naming conventions

### PR DESCRIPTION
**Pull request by BenjaminRuston**
## Description

The TROPICS and COWVR converters need updates to the new naming conventions and TEMPEST should be added

### Issue(s) addressed

- fixes #1185

### CODE REVIEW

File: src/hdf5/cowvr_hdf5_2ioda.py [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+        tic = time.perf_counter()
+        if print_log: print(f"  ... starting timer: {tic:0.3f}")
+        return tic
+    else:
+        toc = time.perf_counter()
+        if print_log: print(f"  ... elapsed time (sec): {toc - tic:0.3f}")
+        return toc
+
+
+def concat_obs_dict(obs_data, append_obs_data):
```
Comment (BenjaminRuston) [inline src/hdf5/cowvr_hdf5_2ioda.py:396]: need to import this from the lib_python.jedi_utils too

**Review by fcvdb — APPROVED** [review]
I tested the new converters on files:
/work2/noaa/da/bruston/data_repos/cowvr/COWVR_TSDR.001775.20220315T235845.20220316T010345.V734.J.20220904T005500.h5
and
/work2/noaa/da/bruston/data_repos/tropics/20220615/TROPICS01.BRTT.L1B.Orbit05329.V02-03.ST20220615-235247.ET20220616-012753.CT20220616-194947.nc
And obtained two IODA files.

**Comment by BenjaminRuston** [comment]
@huishao-r ok this is ready it updates to the new conventions and adds capability for COWVR and TEMPEST still some consolidation to be done.

**Review by fcvdb — APPROVED** [review]
I haven't tested but the changes look good.

**Review by huishao-r — APPROVED** [review]

**Review by PatNichols — APPROVED** [review]
@BenjaminRuston Looks good.

**Pull request merged**